pkgs/development/compilers/zig/0.9.1.nix

```diff
@@ -74,6 +74,11 @@ stdenv.mkDerivation rec {
     runHook postCheck
   '';
 
+  passthru.buildZigPackage = import ../../../build-support/zig/build-zig-package/default.nix {
```

Comment:
Except maybe this.

Reply:
How would I export the zig version with finalAttrs?